web/src/pages/ManageAccountsPage.module.css

```diff
@@ -0,0 +1,239 @@
+.title {
+    composes: h2 from '../App.css';
+    text-align: left;
+    margin-top: 94px;
+    margin-left: 12px;
```

Comment:
Any reason for the left margin here? In the mockup the title looks aligned all the way to the left